lecture_2/homework/app/serializers/clan_serializer.rb

```diff
@@ -0,0 +1,15 @@
+class ClanSerializer < ApplicationSerializer
+
+  # The type parameters is optional in the docs, but I'm thinking how should it be implemented with an uppercase letter in front like a model? 
+  set_type :Clan
```

Comment:
Standard tego nie specyfikuje, ale we wszystkich [przykładach](https://jsonapi.org/format/#document-resource-objects) używają wariantu małymi literami w liczbie mnogiej. Raczej nie ma potrzeby tego zmieniać